### PR TITLE
Add Db models, migrations and fix DbOption properties and naming

### PR DESCRIPTION
**Pull request by AlfaRedDog**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/1134

### CODE REVIEW

**Review by Levliv — APPROVED** [review]

**Review by Levliv — COMMENTED** [review]
Is it possible to use file-scoped namespaces in 
src/SurveyService.Data.Provider/IDataProvider.cs
src/SurveyService.Data.Provider.MsSql.Ef/SurveyServiceDbContext.cs
?

**Review by maybee16 — APPROVED** [review]

File: src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs [hunk 1]
```diff
@@ -42,6 +42,7 @@ protected override void Up(MigrationBuilder builder)
       columns: table => new
```
Comment (vorobevamv) [inline src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs:42]: add columns: IsActive, ModifiedBy, ModifiedAtUtc

File: src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs [hunk 2]
```diff
@@ -42,6 +42,7 @@ protected override void Up(MigrationBuilder builder)
...
       {
         Id = table.Column<Guid>(nullable: false),
+        QuestionId = table.Column<Guid>(nullable: false),
         Content = table.Column<string>(nullable: false),
         CreatedBy = table.Column<Guid>(nullable: false),
         CreatedAtUtc = table.Column<DateTime>(nullable: false),
```
Comment (vorobevamv) [inline src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs:50]: please, move "IsCustom" to line before "CreatedBy" (Created.. and Modified.. usually are in the end of model)

File: src/SurveyService.Models.Db/DbOption.cs [hunk 3]
```diff
@@ -10,21 +10,22 @@ public class DbOption
   public const string TableName = "Options";
 
   public Guid Id { get; set; }
+  public Guid QuestionId { get; set; }
   public string Content { get; set; }
   public bool IsCustom { get; set; }
```
Comment (vorobevamv) [inline src/SurveyService.Models.Db/DbOption.cs:15]: add IsActive

File: src/SurveyService.Models.Db/DbOption.cs [hunk 4]
```diff
@@ -10,21 +10,22 @@ public class DbOption
...
   public Guid CreatedBy { get; set; }
   public DateTime CreatedAtUtc { get; set; }
```
Comment (vorobevamv) [inline src/SurveyService.Models.Db/DbOption.cs:18]: add ModifiedBy, ModifiedAtUtc

File: src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs (hunk 1 above)
Comment (AlfaRedDog) [reply src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs:42]: Done

File: src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs (hunk 2 above)
Comment (AlfaRedDog) [reply src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs:50]: Done

File: src/SurveyService.Models.Db/DbOption.cs (hunk 3 above)
Comment (AlfaRedDog) [reply src/SurveyService.Models.Db/DbOption.cs:15]: Done

File: src/SurveyService.Models.Db/DbOption.cs (hunk 4 above)
Comment (AlfaRedDog) [reply src/SurveyService.Models.Db/DbOption.cs:18]: Done

**Review by Vasis3038 — APPROVED** [review]

File: src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs [hunk 5]
```diff
@@ -42,14 +42,50 @@ protected override void Up(MigrationBuilder builder)
...
         Id = table.Column<Guid>(nullable: false),
+        QuestionId = table.Column<Guid>(nullable: false),
         Content = table.Column<string>(nullable: false),
+        IsCustom = table.Column<bool>(nullable: false),
         CreatedBy = table.Column<Guid>(nullable: false),
         CreatedAtUtc = table.Column<DateTime>(nullable: false),
-        IsCustom = table.Column<bool>(nullable: false)
+        ModifiedBy = table.Column<Guid>(nullable: true),
+        ModifiedAtUtc = table.Column<DateTime>(nullable: true),
+        IsActive = table.Column<bool>(nullable: false)
```
Comment (vorobevamv) [inline src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs]: please, move IsActive to line before CreatedBy

Comment (AlfaRedDog) [reply src/SurveyService.Data.Provider.MsSql.Ef/Migrations/20220826180800_InitialCreate.cs]: Done

**Review by vorobevamv — APPROVED** [review]

**Pull request merged**